<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MWindow</class>
 <widget class="QMainWindow" name="MWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>721</width>
      <height>401</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="pgBienvenida">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>60</y>
        <width>351</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Bienvenido</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>190</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">pressed{
	color:blue;
}</string>
      </property>
      <property name="text">
       <string>Nueva Tabla...</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>230</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">pressed{
	color:blue;
}</string>
      </property>
      <property name="text">
       <string>Abrir Tabla...</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pgAdicionCampos">
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>20</y>
        <width>341</width>
        <height>371</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Tipo</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Longitud</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nombre</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>290</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Tipo</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>140</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Longitud</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>80</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Entero</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Texto</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>140</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>190</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>190</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pgAdicionRegistros">
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>601</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>4</x>
        <y>100</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar Fila</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>742</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
